manifests/agent.pp

```diff
@@ -200,60 +200,64 @@
+  $logremotecommands                      = $zabbix::params::agent_logremotecommands,
+  $server                                 = $zabbix::params::agent_server,
+  $listenport                             = $zabbix::params::agent_listenport,
+  $listenip                               = $zabbix::params::agent_listenip,
+  $startagents                            = $zabbix::params::agent_startagents,
+  $serveractive                           = $zabbix::params::agent_serveractive,
+  $hostname                               = $zabbix::params::agent_hostname,
+  $hostnameitem                           = $zabbix::params::agent_hostnameitem,
+  $hostmetadata                           = $zabbix::params::agent_hostmetadata,
+  $hostmetadataitem                       = $zabbix::params::agent_hostmetadataitem,
+  $refreshactivechecks                    = $zabbix::params::agent_refreshactivechecks,
+  $buffersend                             = $zabbix::params::agent_buffersend,
+  $buffersize                             = $zabbix::params::agent_buffersize,
+  $maxlinespersecond                      = $zabbix::params::agent_maxlinespersecond,
+  $zabbix_alias                           = $zabbix::params::agent_zabbix_alias,
+  $timeout                                = $zabbix::params::agent_timeout,
+  $allowroot                              = $zabbix::params::agent_allowroot,
+  $zabbix_user                            = $zabbix::params::agent_zabbix_user,
+  $include_dir                            = $zabbix::params::agent_include,
+  $include_dir_purge                      = $zabbix::params::agent_include_purge,
+  $unsafeuserparameters                   = $zabbix::params::agent_unsafeuserparameters,
+  $userparameter                          = $zabbix::params::agent_userparameter,
+  $loadmodulepath                         = $zabbix::params::agent_loadmodulepath,
+  $loadmodule                             = $zabbix::params::agent_loadmodule,
+  $tlsaccept                              = $zabbix::params::agent_tlsaccept,
+  $tlscafile                              = $zabbix::params::agent_tlscafile,
+  $tlscertfile                            = $zabbix::params::agent_tlscertfile,
+  $tlsconnect                             = $zabbix::params::agent_tlsconnect,
+  $tlscrlfile                             = $zabbix::params::agent_tlscrlfile,
+  $tlskeyfile                             = $zabbix::params::agent_tlskeyfile,
+  $tlspskfile                             = $zabbix::params::agent_tlspskfile,
+  $tlspskidentity                         = $zabbix::params::agent_tlspskidentity,
+  $tlsservercertissuer                    = $zabbix::params::agent_tlsservercertissuer,
+  $tlsservercertsubject                   = $zabbix::params::agent_tlsservercertsubject,
+  String $agent_config_owner              = $zabbix::params::agent_config_owner,
+  String $agent_config_group              = $zabbix::params::agent_config_group,
```

Comment:
thats why I throw them in their own commit: https://github.com/voxpupuli/puppet-zabbix/pull/382/commits/8bdcaddb246781f8c77d8a008d823e83400dedfd